ligo/src/types.mligo

```diff
@@ -21,6 +21,18 @@ type ledger_key = address * token_id
 type ledger_value = nat
 type ledger = (ledger_key, ledger_value) big_map
```

Comment:
Note: I won't suggest modifying the type (as one may want to store other tokens and there's another issue to support that as well), however perhaps we should no longer use the ledger to store frozen tokens as they are now tracked in the `freeze_history` and having it here as well could leave more space for mistakes.